src/main/resources/templates/fragments/organisation.html

```diff
@@ -3,8 +3,7 @@
 <html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <body>
   <th:block th:fragment="organisation(organisation, branding)">
-    <span th:if="${branding.present}" th:utext="${#strings.replace(branding.get().logoFormattedName,'\r\n','&lt;br/&gt;')}" class="organisation-logo" th:classappend="|organisation-logo-${branding.get().logoClassName} ${branding.get().colourClassName}|">Cabinet Office</span>
-    <span th:unless="${branding.present}" th:text="${organisation.name}" class="organisation-logo organisation-logo-no-identity">Cabinet Office</span>
+    <span th:text="${organisation.name}">Cabinet Office</span>
```

Comment:
Can we delete this file? It seems a bit overkill to have it as a fragment now as it's only being used in one place.

Reply:
Yes, agreed. I need your help